src/core/radio/radio.hpp

```diff
@@ -752,7 +752,10 @@ inline void Radio::GetIeeeEui64(Mac::ExtAddress &aIeeeEui64)
     otPlatRadioGetIeeeEui64(GetInstancePtr(), aIeeeEui64.m8);
 }
 
-inline uint32_t Radio::GetSupportedChannelMask(void) { return otPlatRadioGetSupportedChannelMask(GetInstancePtr()); }
+inline uint32_t Radio::GetSupportedChannelMask(void) const
+{
+    return otPlatRadioGetSupportedChannelMask(GetInstancePtr());
```

Comment:
In case of radio is implemented as RCP, does this mean each time we want to get the mask we need go to RCP? 
It may not be a good model if we need to do this.